pyproject.toml

```
@@ -99,12 +99,17 @@ rosa = []
 #     Tornado 6.
 #     tornado
 # ]
+rose-edit = [
```

(needs to match the `metomi-rose[edit]` below)

```suggestion
edit = [
```

---

IMO @oliver-sanders I'd be happy for us to merge this.

---

Done.